tests/Unit/DSC_SqlSetup.Tests.ps1

```diff
@@ -2707,6 +2823,52 @@ Describe 'SqlSetup\Set-TargetResource' -Tag 'Set' {
             }
         }
 
+        Context 'When installing the database engine and ProductcoveredBySA is true' {
```

Comment:
I'm not in front of a big enough screen to look at the whole test file, but this test looks to be a copy + paste of the test added at line 2484. Is this supposed to be here?

Reply:
ah, line 2826 is the actual test, I don't know how 2484 got there, I should have misplaced it somehow. let me remove it

Reply:
@dan-hughes all set I guess -):